res/controllers/Denon-Prime-4-scripts.js

```diff
@@ -119,6 +193,35 @@ Prime4.Deck = function(deckNumbers, midiChannel) {
 
 Prime4.Deck.prototype = new components.Deck();
 
+// Headphone Cue / PFL buttons for Decks 1 and 2
+
+// eslint-disable-next-line-no-var
+var headphoneCue = [];
```

Comment:
Please don't disable eslint for no reason. 
```suggestion
let headphoneCue = [];
```